Hackney.Core/Hackney.Core.Testing/Hackney.Core.Testing.Shared/JsonOptions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Hackney.Core.Testing
+{
+    public static class JsonOptions
```

Comment:
This class should probably not go here but in a common functional package, not one specifically for testing.